optaplanner-build-parent/pom.xml

```diff
@@ -759,8 +759,11 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
+          <version>3.2.0</version>
```

Comment:
To get `<failOnWarnings>`.

Reply:
We inherit the maven-javadoc-plugin from jboss-parent, so we would set the version with a property `<version.javadoc.plugin>3.2.0</>` too. Not saying we should, but note that we do do that for surefire/failsafe sometimes... (=> consistency dilemma)